cli/Cargo.toml

```diff
@@ -55,10 +55,6 @@ experimental = [
     "circuit",
     "health",
     "database",
-    "database-migrate-biome-credentials",
```

Comment:
There should be a single `"database-migrate-biome"` feature, as biome is now the component boundary.  When splinter admin state is in a database, it will have an equivalent `"database-migrate-admin"` which can be enabled separately from the biome migrations.

Reply:
Added database-migrate-biome feature to cli